test/tcollector-reporter.js

```diff
@@ -152,9 +152,9 @@ test('jsonSpanToThriftSpan', function t2(assert) {
     var mapped = TCollectorReporter.jsonSpanToThriftSpan(testSpan);
 
     assert.equals(mapped.name, testSpan.name);
-    assert.deepEquals(mapped.traceId, testSpan.traceid);
-    assert.deepEquals(mapped.id, testSpan.id);
-    assert.deepEquals(mapped.parentId, testSpan.parentid);
+    assert.deepEquals(mapped.traceId, Buffer([0, 0, 0, 0, 0, 0, 0, 1]));
```

Comment:
These are about to be thrift serialized, and in the thrift file these are still `binary`. `jsonSpanToThriftSpan` converts a span from the internal TChannel representation to something ready to be sent to the `tchannelThrift.send` call.